manipulation/src/functTest/resources/elasticsearch/buildSrc/src/main/minimumRuntime/org/elasticsearch/gradle/VersionProperties.java

```diff
@@ -35,17 +35,19 @@ public static String getLucene() {
     }
 
     private static Properties getVersionProperties() {
-        Properties props = new Properties();
-        InputStream propsStream = VersionProperties.class.getResourceAsStream("/version.properties");
-        if (propsStream == null) {
-            throw new IllegalStateException("/version.properties resource missing");
-        }
-        try {
-            props.load(propsStream);
+        Properties versionProps = new Properties();
+
+        try (InputStream propsStream = VersionProperties.class.getResourceAsStream("/version.properties")) {
+            if (propsStream == null) {
+                throw new IllegalStateException("/version.properties resource missing");
+            }
+
+            versionProps.load(propsStream);
         } catch (IOException e) {
             throw new IllegalStateException("Failed to load version properties", e);
         }
-        return props;
+
+        return versionProps;
```

Comment:
You could make a PR to elasticsearch if that code is still the same - that is where that test case was sourced from!

Reply:
I will look. I also changed the name of the `props` variable as you saw. I think it was shadowing a field of the same name which was a little confusing.